src/app/services/category.service.ts

```diff
@@ -13,25 +13,31 @@ export class CategoryService {
 
   addNewCategory(category: ICategory): Observable<ICategory> {
     return this.httpService
-      .post<ICategory>(category, 'Category')
+      .post<ICategory>(category, RouteEndpoints.CATEGORY)
       .pipe(catchError(this.httpService.handleHttpErrorResponse));
   }
 
   deleteCategory(id: number): Observable<ICategory> {
     return this.httpService
-      .delete<ICategory>(id, `Category?categoryId=`)
-      .pipe(catchError(this.httpService.handleHttpErrorResponse));
+		.delete<ICategory>(id, RouteEndpoints.CATEGORY+`?categoryId=`)
+		.pipe(catchError(this.httpService.handleHttpErrorResponse));
   }
 
   getCategory(id: number): Observable<ICategory> {
     return this.httpService
-      .getById<ICategory>(id, 'Category')
-      .pipe(map((response) => response.result), catchError(this.httpService.handleHttpErrorResponse));
+		.getById<ICategory>(id, RouteEndpoints.CATEGORY)
+		.pipe(
+			map((response) => response.result),
+			catchError(this.httpService.handleHttpErrorResponse)
+		);
   }
 
   updateCategory(category: ICategory): Observable<ICategory> {
     return this.httpService
-      .update<ICategory>(category, `Category?categoryId=${category.id}`)
-      .pipe(catchError(this.httpService.handleHttpErrorResponse));
+		.update<ICategory>(
+			category,
+			RouteEndpoints.CATEGORY+`?categoryId=${category.id}`
```

Comment:
RouteEndpoints.CATEGORY can also be part of the template string, like this:

${RouteEndpoints.CATEGORY}?categoryId=${category.id}